src/pykeen/lr_schedulers/__init__.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+
+"""Learning Rate Schedulers available in PyKEEN.
+
+===========================  =============================================================
```

Comment:
This might not be necessary, I sort of discontinued doing this on the other components since I had the sphinx-automodapi start doing it for me